src/posix.mak

```diff
@@ -137,7 +137,7 @@ else
   # Auto-bootstrapping, will download dmd automatically
   # Keep var below in sync with other occurrences of that variable, e.g. in circleci.sh
   HOST_DMD_VER=2.072.2
-  HOST_DMD_ROOT=$(TMP)/.host_dmd-$(HOST_DMD_VER)
+  HOST_DMD_ROOT=$(GENERATED)/host_dmd-$(HOST_DMD_VER)
```

Comment:
`make clean` deletes the directory. This means that every time you do it you will have to download the bootstrapping version. I suggest you choose a different directory (or create one) so that the compiler is downloaded only once.

Reply:
I'm aware, but
- remember that you don't need to call `clean` because `make` is smart enough
- `$(TMP)` gets nuked too
- I don't think anyone developing on the compiler doesn't have a DMD compiler installed
- If I create a new top-level directory, people will complain too
- If I change `clean` to only nuke `generated/$(OS)` people will complain too
- we do this already for `dlang.org` ([source](https://github.com/dlang/dlang.org/blob/master/posix.mak#L185))

That being said, I honestly don't care which path is taken except that it can't be one outside of this directory as this will lead to CI errors when running in parallel.

Reply:
Fair enough